cylc/flow/cfgspec/globalcfg.py

```diff
@@ -990,12 +990,26 @@
                    the wrapper script.
             ''')
             Conf('global init-script', VDR.V_STRING, desc='''
-                Add a script before the init-script of all jobs on this
-                platform.
+                A per-platform script which is run before the job file is
+                sourced.
```

Comment:
Needs explaining in user-friendly terms.  I guess you mean, *before `etc/job.sh` is sourced*?  If so, that's not something users need to know about. They'd likely interpret it as "before the task job script is sourced" (and that isn't sourced).

Maybe something like this?:
```suggestion
                A per-platform script which is run before any other task job scripting.
```